MovieCave - MVVM/Movie-Cave/Movie-Cave/View/AddMovieView/AddMovieViewController.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  AddMovieViewController.swift
+//  Movie-Cave
+//
+//  Created by Admin on 7.09.23.
+//
+
+import UIKit
+import Combine
+
+class AddMovieViewController: UIViewController {
+    
+    //MARK: - IBOutlets
+    @IBOutlet private weak var genreCheckView: UIView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var contentViewHeight: NSLayoutConstraint!
+    @IBOutlet private weak var raitingView: UIView!
+    @IBOutlet private weak var genreView: UIView!
+    @IBOutlet private weak var doneButton: UIButton!
+    @IBOutlet private weak var movieImageButton: UIButton!
+    @IBOutlet private weak var movieTitleTextField: UITextField!
+    @IBOutlet private weak var describtionTextField: UITextField!
+    
+    //MARK: - Properties
+    var movieCompletion: ((MovieData) -> Void)?
+    private var coreData: CoreDataManager = CoreDataManager.shared
```

Comment:
Same dependency witch is used in the root view controller is used here, try to find a way to inject it here without instantiate it here.